src/servers/ZoneServer2016/entities/character.ts

```diff
@@ -367,8 +389,31 @@ export class Character2016 extends BaseFullCharacter {
         desiredEnergyIndicator = "";
         break;
     }
+
+    let desiredComfortIndicator = "";
+    const comfortIndicators = ["COMFORT +", "COMFORT ++"];
+    switch (true) {
+      case comfort > 2001:
+        desiredComfortIndicator = "COMFORT ++";
+        client.character._resources[ResourceIds.HEALTH] += 100;
```

Comment:
Health regen just by having comfort ? I don't remember how it was.

Reply:
I mistakenly read on the article that there was a small amount of passive regen, but in reality it's more like an extremely small health passive regen rather than being noticeable. https://www.youtube.com/watch?v=mUm95wfBj94&t=5456s
I watched this from the timestamp to the end and his health went up a few while having comfort++ so there is a tiny tiny passive health regen with comfort+ and comfort++, so I modified it to being closer to that.